Ollivanders/src/net/pottercraft/Ollivanders2/Effect/HERBIFORS.java

```diff
@@ -16,6 +16,7 @@
  * @version Ollivanders2
  * @author lownes
  */
+@Deprecated
```

Comment:
Why are we deprecating herbifors?

Reply:
The Effect is no longer used since Herbifors was refactors in June 2017.